compile.sh

```diff
@@ -1,12 +1,18 @@
 #!/bin/bash
 set -e # Exit with nonzero exit code if anything fails
 
-echo 'Building spec'
-make -C spec
-echo 'Building wgsl'
-make -C wgsl
-echo 'Building explainer'
-make -C explainer
+# Compile specs by default, unless told to only copy static files
+if [ $1 == 'static' ]; then
+  echo 'Extracting IDL from WebGPU spec'
+  make -C spec webgpu.idl
+else
```

Comment:
Seeing how this is used, it looks like lines 8 through 14 should not conditionally executed.

Reply:
Does my previous comment clarify why I don't want the HTML files to be generated when the script is called with `static`?